src/VDB/Spider/Spider.php

```diff
@@ -2,10 +2,10 @@
 
 namespace VDB\Spider;
 
-use Symfony\Component\EventDispatcher\Event;
 use Symfony\Component\EventDispatcher\EventDispatcher;
 use Symfony\Component\EventDispatcher\EventDispatcherInterface;
 use Symfony\Component\EventDispatcher\GenericEvent;
+use Symfony\Contracts\EventDispatcher\Event;
```

Comment:
Ditto

Reply:
Yes, you are right - these shouldn't be needed anymore. I've removed them and update the PR @mvdbos